src/components/Molecules/Promo/Promo.js

```diff
@@ -5,53 +5,50 @@ import Picture from '../../Atoms/Picture/Picture';
 import {
   Container,
   NudgeBottom,
```

Comment:
Did you try clip-path ?

Reply:
No, clip-path is interesting .. looks like it has good support .. I will try it out!

Reply:
Can you update font family of CTA as well